src/pages/PreviousRuns.js

```diff
@@ -60,21 +63,31 @@ export const PreviousRuns = () => {
             hoverHighlight: true,
             columns: [
               {
-                size: { basis: 300 },
-                field: 'workflowName',
-                headerRenderer: () => h(Sortable, { sort, field: 'workflowName', onSort: setSort }, ['Workflow Name']),
+                size: { basis: 350 },
+                field: 'runId',
+                headerRenderer: () => h(Sortable, { sort, field: 'runId', onSort: setSort }, ['Run ID']),
                 cellRenderer: ({ rowIndex }) => {
-                  return h(TextCell, [paginatedPreviousRuns[rowIndex].name])
+                  return h(TextCell, [paginatedPreviousRuns[rowIndex].run_id])
```

Comment:
same comment here about snake vs. camel case

Reply:
As Adam said, since CBAS returns `run_id` and this is extracting keys from the response JSON object, it needs to be the same. If we use `runId` here, it doesn't find any such key in the `paginatedPreviousRuns` and displays nothing on the UI.